src/components/Main/MovieCard.jsx

```diff
@@ -53,43 +54,40 @@ const MovieCard = ({
         return null;
     }
   };
-
   return (
     <React.Fragment>
       <Modal modalState={editModal} setModalState={setEditModal}>
-        <AddModal modalTitle="Edit" setModalState={setEditModal} />
+        <AddModal
+          modalTitle="Edit"
+          setModalState={setEditModal}
+          movieId={id}
+          setSrc={setSrc}
+        />
       </Modal>
       <Modal modalState={deleteModal} setModalState={setDeleteModal}>
         <NotificationModal
           title="Delete"
           info="Are you sure you want to delete this movie?"
           button="Delete"
           setDeleteModal={setDeleteModal}
+          movieId={id}
+          setSrc={setSrc}
         />
       </Modal>
 
       <div
         className="moviecard"
         onClick={() => {
-          dispatch(
-            clickMovie({
-              id,
-              title,
-              poster_path,
-              genres,
-              release_date,
-              overview,
-              vote_average,
-              runtime,
-            })
-          );
+          navigate(`/search/${title}/${id}`);
```

Comment:
I am seeing many places with "./search/ title or /search / something lese". Do you think if there are places where the path are repeated can be saved as a constant in one place ?

Reply:
Created constant variable for this case in directory  /src/dictionary/index.js  .   Fixed